serverless.ts

```diff
@@ -479,10 +495,19 @@ const serverlessConfig: AWS = {
       },
       DataBackupBucket: {
         Type: 'AWS::S3::Bucket',
-        Condition: 'IsDev',
+        Condition: 'IsDevOrProd',
         DeletionPolicy: 'Retain',
         Properties: {
           BucketName: '${self:service}-${self:provider.stage}-data-backup',
+          AccessControl: 'Private',
+          LifecycleConfiguration: {
+            Rules: [
+              {
+                Id: 'delete-after-3-months',
+                ExpirationInDays: 90,
```

Comment:
do we want the same rules for prod and dev?

Reply:
I think it's better they share these settings than just having expiration for prod

Reply:
For now I don't mind them sharing the expiration date, but we can split in the future